src/internal/models/post_models.go

```diff
@@ -11,7 +11,8 @@ type PostDB struct {
 }
 
 type PostInput struct {
-	UserId	 string `bson:"user_id"`
+	Id        string `bson:"_id"`
```

Comment:
Varför har input id? Creating posts tar in input, och då har man inget id.
Skulle en model postCreate vara med passande?

Reply:
Behöver ha en struct som innehåller ID så det kan sättas innan det skickas till databasen, så den behöver inte sättas innan den skickas in till delarna jag skrivit nu.  Alternativt att vi gör en till struct som har ID också, men känns onödigt att skapa en till som vi måste omvandla till